client/my-sites/checkout/checkout/domain-refund-policy.jsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { gaRecordEvent } from 'lib/analytics/ga';
+import { DOMAIN_CANCEL, REFUNDS } from 'lib/url/support';
+import Gridicon from 'components/gridicon';
+import {
+	hasDomainBeingUsedForPlan,
+	hasDomainRegistration,
+	hasDomainRenewal,
+	hasNewDomainRegistration,
+} from 'lib/cart-values/cart-items';
+
+class DomainRefundPolicy extends React.Component {
+	static displayName = 'RegistrationRefundPolicy';
+
+	recordCancelDomainSupportClick = () => {
+		gaRecordEvent( 'Upgrades', 'Clicked Cancel Domain Support Link' );
+	};
+
+	recordRefundsSupportClick = () => {
+		gaRecordEvent( 'Upgrades', 'Clicked Refund Support Link' );
+	};
+
+	/**
+	 * @returns {[]} Returns an array of renderable policies.
+	 */
+	getApplicablePolicies = () => {
+		const policies = [];
+
+		if ( hasNewDomainRegistration( this.props.cart ) ) {
+			policies.push( this.renderNewPolicy() );
+		}
+
+		if ( hasDomainRenewal( this.props.cart ) ) {
+			policies.push( this.renderRenewalPolicy() );
+		}
+
+		return policies;
+	};
+
+	renderNewPolicy = () => {
+		const refundsSupportPage = (
+			<a
+				href={ REFUNDS }
+				target="_blank"
+				rel="noopener noreferrer"
+				onClick={ this.recordRefundsSupportClick }
+			/>
+		);
+
+		let message = this.props.translate(
+			'You understand that {{refundsSupportPage}}domain name refunds{{/refundsSupportPage}} are limited to 96 hours after registration.',
+			{
+				components: {
+					refundsSupportPage: refundsSupportPage,
+				},
+			}
+		);
+
+		if ( hasDomainBeingUsedForPlan( this.props.cart ) ) {
+			message = this.props.translate(
+				'You understand that {{refundsSupportPage}}domain name refunds{{/refundsSupportPage}} are limited to 96 hours after registration. Refunds of paid plans will deduct the standard cost of any domain name registered within a plan.',
```

Comment:
Nice addition! Though, I'm not sure about the wording: `any domain name registered within a plan.`. Did you mean `with a plan`? Also, hopefully the user will understand that that applies also to the domain credit coming from the plan - but I don't think we need to say that explicitly.

Reply:
This wasn't me! 😁 This was already in the `DomainRegistrationRefundPolicy` code I deleted. So for now, I don't want to change that in this PR -- we can flag a revisit separately.